toolbox/fdc3-compliance/src/test/fdc3.getInfo.ts

```diff
@@ -0,0 +1,5 @@
+describe("fdc3.getInfo", () => {
+  it("Method is callable", async () => {
+    window.fdc3.getInfo();
```

Comment:
We could add an await, as it won't break anything and won't require a revisit for 2.0.